pkg/activator/handler/handler.go

```diff
@@ -65,6 +65,7 @@ type activationHandler struct {
 	sksLister      netlisters.ServerlessServiceLister
 }
 
+var IsTraceEnabled = false
```

Comment:
I know this is WIP but one comment on the approach: A global variable is not good for a whole slew of reasons:

- Hard to test
- Easy to get wrong
- Breaks the flow of the program
- ...

Instead, could we surface both `transport` and `probeTransport` as arguments? (I guess at this point we might as well think about ditching the `New` method and build the struct straight away 😆 ).

Alternatively, we could pass a `transportWrapper` function, which optionally wraps a transport, but that seems an unnecessary abstraction for now.

Reply:
Yep - It's marked as WIP specifically because of this. 👍 I'm still figuring out how to inform the handler about changes to tracing when a change is made to the config store.

Reply:
We chatted about this offline, but I think you can get this value by pulling the activator config store out of the request context and accessing store.Tracing.IsEnabled